### PR TITLE
Refined WebSocket Message Architecture

### LINKED ISSUE #25: Refine WebSocket Message Architecture
Acceptance Criteria:
- A Game Class Exists
- Message Structure and Validation exists
- One player can connect and configure
- Second player can connect

### LINKED ISSUE #35: Fix Python App so it exits cleanly

### PR DESCRIPTION
**Pull request by walshb421**
closes #25
closes #35

### CODE REVIEW

File: engine/app.py [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
+import json
+
+#from board import ChessBoard
 
 # Runs when websocket connection is established
 async def main(websocket):
     # Initalize the board
-    board = ChessBoard()
-    board.create_starting_board()    
+    game = Game()
```
Comment (GarettOS) [inline engine/app.py:11]: ;)

**Review by GarettOS — COMMENTED** [review]
Nice this looks good, just needs to be implemented to work with the board

**Comment by walshb421** [comment]
> Nice this looks good, just needs to be implemented to work with the board

Thanks for the feedback.

This was supposed to be a draft still

**Review by GarettOS — APPROVED** [review]
I like I like game clean blends with the board now

**Comment by walshb421** [comment]
> I like I like game clean blends with the board now

Yeah and I believe this will make it easier to make architectural changes without affecting your chess board too much

**Pull request merged**